### PR TITLE
Update LegitimateURLShortener.txt

### PR DESCRIPTION
**Pull request by shenzhiming88**
Soft it simply.

### CODE REVIEW

**Comment by DandelionSprout** [comment]
I feel `! ——— General block rules ——— !` should be below `! ——— General remove rules ——— !`, and not above it.

File: LegitimateURLShortener.txt [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+! Version: 02November2021v2-Beta
 ! Expires: 2 days
 ! Description: In a world dominated by bit.ly, ad.fly, and several thousand other malware cover-up tools, this list reduces the length of URLs in a much more legitimate and transparent manner. Essentially, it automatically removes unnecessary $/& values from the URLs, making them easier to copy from the URL bar and pasting elsewhere as links. Enjoy.
 ! Homepage: https://github.com/DandelionSprout/adfilt/discussions/163
 ! Homepage for the repository: https://github.com/DandelionSprout/adfilt/blob/master/Wiki/General-info.md#english
 ! Opt-in allowlist for affiliate support tags: https://raw.githubusercontent.com/DandelionSprout/adfilt/master/LegitimateURLShortener-AffiliateTagAllowlist.txt
 ! Special thanks to https://github.com/git-101-collab for providing a 3-digit amount of new entries to this list, and to https://github.com/iam-py-test for volunteering to speed up the addition of user-submitted entries.
 
-! ——— General rules ——— !
+! ——— General block rules ——— !
```
Comment (iam-py-test) [inline LegitimateURLShortener.txt:10]:
```suggestion
! ——— General blocking rules ——— !
```
I feel this should be _Blocking_ but that's just my opinion.

File: LegitimateURLShortener.txt [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
+!! Scripts that solely exist to make URLs longer
+! https://github.com/the1812/Bilibili-Evolved/issues/2247
```
Comment (iam-py-test) [inline LegitimateURLShortener.txt:12]: I feel this should have a link to the PR where it was added (#320)

File: LegitimateURLShortener.txt [hunk 3]
```diff
@@ -878,7 +880,7 @@ $removeparam=ceneo_spo
 ^from=ceneo^$removeparam=from
 ^ref=ceneo^$removeparam=ref
 
-! ——— Specific rules ——— !
+! ——— Specific remove rules ——— !
```
Comment (iam-py-test) [inline LegitimateURLShortener.txt:883]: I agree with Imre

**Review by iam-py-test — CHANGES_REQUESTED** [review]
Thank you for opening this Pull Request

**Review by iam-py-test — APPROVED** [review]
This looks great! Thank you
Merging...

**Pull request merged**